pkg/ibmvpc/vpc.go

```diff
@@ -39,8 +39,8 @@ type ReservedIP struct {
 
 func (r *ReservedIP) NameForAnalyzerOut(c *vpcmodel.VPCConfig) string {
 	prefix := ""
-	if c != nil && c.IsMultipleVPCsConfig {
-		prefix = r.VPC().Name() + vpcmodel.Deliminator
+	if c != nil {
+		prefix = c.MultipleVPCsConfigPrefix(r.VPC().Name())
```

Comment:
can add another internal function that gets as input arg the config object and returns that prefix, and here always just have `prefix + nameWithBracketsInfo(r.vpe, r.Address())`
(to avoid this duplication across this file as well) .